components/SearchPlaces/SearchPlaces.js

```diff
@@ -0,0 +1,17 @@
+import {
+  SearchInput,
+  SearchButton,
+  SearchForm,
+  MagnifyingGlass,
+} from "./StyledSearchPlaces";
+
+export default function SearchPlaces({ handleSearchSubmit }) {
+  return (
+    <SearchForm onSubmit={(event) => handleSearchSubmit(event)}>
+      <SearchButton type="submit">
+        <MagnifyingGlass />
+      </SearchButton>
+      <SearchInput type="text" placeholder="Search for places" />
```

Comment:
You should add a `name` attribute and an `aria-label` since the input field misses a corresponding label.

Reply:
```suggestion
        <SearchForm onSubmit={(event) => onSearchSubmit(event)}>
      <SearchButton aria-label="search-button" type="submit">
        <MagnifyingGlass />
      </SearchButton>
      <SearchInput
        type="text"
        placeholder="Search for places"
        name="search-destination"
      />
    </SearchForm>
```
Is that correct?

Reply:
I would also be great to have an `aria-label` for the search input field.
But apart from that everything else is fine 👍